website/content/en/docs/deployment/prerequisites.md

```diff
@@ -81,5 +81,28 @@ The command above installs the following tools:
 - [terraform](https://learn.hashicorp.com/tutorials/terraform/install-cli) - An infrastructure as code tool that lets you develop cloud and on-prem resources.
 - [helm](https://helm.sh/docs/intro/install/) - A package manager for Kubernetes
 
+## Configure AWS CLI
+
+To access AWS services with the AWS CLI, you need an AWS account, IAM credentials, and an IAM access key pair. When running AWS CLI commands, the AWS CLI needs to have access to those AWS credentials. 
+
+Note that your account must have **administrative privileges**. 
+
+Run the following command to configure AWS CLI:
```

Comment:
Lets suggest this link for quick configuration https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-quickstart.html#cli-configure-quickstart-config instead of direct instructions because the guide is pretty detailed